manifests/deploy/upstream/olm/metering.catalogsource.yaml

```diff
@@ -1,10 +1,10 @@
-apiVersion: operators.coreos.com/v1
+apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
-  name: installed-upstream-metering-operators-metering
-  namespace: marketplace
+  name: installed-redhat-metering-operators-openshift-metering
+  namespace: openshift-marketplace
 spec:
-  targetNamespace: metering
-  packages: metering-upstream
-  source: upstream-community-operators
+  targetNamespace: openshift-metering
+  sourceType: grpc
+  packages: metering-ocp
```

Comment:
I assume we would want to keep the `packages: metering-upstream` here if we're not using the `upstream-community-operators` `OperatorSource` as a reference point, and instead configuring our own `OperatorSource`.